coriolis/osmorphing/windows.py

```diff
@@ -426,24 +426,24 @@ def _write_cloudbase_init_conf(self, cloudbaseinit_base_dir,
         conf_file_path = "%s\\cloudbase-init.conf" % conf_dir
 
         conf_content = (
-            "[DEFAULT]\n"
-            "username = Admin\n"
-            "groups = Administrators\n"
-            "verbose = true\n"
-            "bsdtar_path = %(bin_path)s\\bsdtar.exe\n"
-            "mtools_path = %(bin_path)s\n"
-            "logdir = %(log_path)s\n"
-            "local_scripts_path = %(scripts_path)s\n"
-            "stop_service_on_exit = false\n"
-            "logfile = cloudbase-init.log\n"
-            "default_log_levels = "
-            "comtypes=INFO,suds=INFO,iso8601=WARN,requests=WARN\n"
-            "allow_reboot = false\n"
-            "plugins = %(plugins)s\n"
-            "debug = true\n"
-            "san_policy = OnlineAll\n"
-            "metadata_services = %(metadata_services)s\n"
-            "logging_serial_port_settings = %(com_port)s,9600,N,8\n" %
+            "[DEFAULT]\r\n"
```

Comment:
Although I can understand the confusion of `Notepad.exe` users when opening non-CRLF files, is there any functional reason for Cloudbase-init to need the `\r`s?

Reply:
No functional reason, besides beautifying. Figured we could write CRLF text files since it's standard on Windows.